### DISCUSSION TITLE
Color-tagged text has strange new line in speech bubble

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15367] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![20241214125857](https://github.com/user-attachments/assets/295add28-d679-427f-b75e-238d8c7f0a89)
![20241214130108](https://github.com/user-attachments/assets/71ed59ea-8c37-48cb-8c0f-38a388c950a4)

### Reproduction steps

There's something alien growing in ‖color:gui.Blue‖Gunnery Compartment‖color:end‖.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Closing discussion as resolved.
